control-plane/connect-inject/endpoints_controller_test.go

```diff
@@ -5804,6 +6009,91 @@ func Test_GetWANData(t *testing.T) {
 	}
 }
 
+func TestCreateTerminatingGatewayRegistrations(t *testing.T) {
```

Comment:
Do we still need this test?

Reply:
No, it's being tested by the "wider scoped" tests.

Reply:
Removed